packages/itwin/property-grid/src/PropertyGridUiItemsProvider.tsx

```diff
@@ -53,10 +53,10 @@ export class PropertyGridUiItemsProvider implements UiItemsProvider {
       )
     ) {
       widgets.push({
```

Comment:
I would like to see  PropertyGridProps expanded to allow the default location to be specified like I am adding in https://github.com/iTwin/viewer-components-react/pull/216.